src/MainWindow.vala

```diff
@@ -354,32 +356,77 @@ namespace Terminal {
             color_button_dark_context.add_class ("color-button");
             color_button_dark_context.add_class ("color-dark");
 
-            var style_popover_grid = new Gtk.Grid ();
-            style_popover_grid.margin = 12;
-            style_popover_grid.column_spacing = 6;
-            style_popover_grid.row_spacing = 12;
-            style_popover_grid.width_request = 200;
-            style_popover_grid.attach (font_size_grid, 0, 0, 3, 1);
-            style_popover_grid.attach (color_button_white, 0, 1, 1, 1);
-            style_popover_grid.attach (color_button_light, 1, 1, 1, 1);
-            style_popover_grid.attach (color_button_dark, 2, 1, 1, 1);
-            style_popover_grid.show_all ();
-
-            var style_popover = new Gtk.Popover (null);
-            style_popover.add (style_popover_grid);
-
-            var style_button = new Gtk.MenuButton ();
-            style_button.set_can_focus (false);
-            style_button.image = new Gtk.Image.from_icon_name ("open-menu-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
-            style_button.popover = style_popover;
-            style_button.tooltip_text = _("Style");
-            style_button.valign = Gtk.Align.CENTER;
+            var color_grid = new Gtk.Grid ();
+            color_grid.column_homogeneous = true;
+            color_grid.margin_start = color_grid.margin_end = 12;
+            color_grid.margin_bottom = 6;
+
+            color_grid.add (color_button_white);
+            color_grid.add (color_button_light);
+            color_grid.add (color_button_dark);
+
+            var natural_copy_paste_label = new Gtk.Label (_("Natural Copy/Paste"));
+            natural_copy_paste_label.halign = Gtk.Align.START;
+            natural_copy_paste_label.hexpand = true;
+
+            var natural_copy_paste_switch = new Gtk.Switch ();
+            natural_copy_paste_switch.halign = Gtk.Align.END;
+
+            var natural_copy_paste_description = new Gtk.Label (_("<small>Shortcuts don’t require Shift, but may interfere with some interactive CLI apps</small>"));
```

Comment:
I think you probably don't want to include the markup in the translatable string